kunstmaan/bundles-cms/4.0/etc/packages/kunstmaan_liip_imagine.yaml

```diff
@@ -0,0 +1,75 @@
+liip_imagine:
```

Comment:
@Devolicious As you're relying on your Standard Edition, maybe you should create your own Flex skeleton?